### PR TITLE
Placeholders made for exercise subpg

### PR DESCRIPTION
**Pull request by bmgdevelopment**
Placeholders made for exercise subpage

### CODE REVIEW

File: pages/resources/counseling.js [hunk 1]
```diff
@@ -3,16 +3,10 @@ import { supabase } from '../../utils/client';
...
 import { counselors } from '../../data/resources';
-import { creativeIconPics } from '../../data/resources';
 import Link from 'next/link'
-import { resourceTagData } from '../../data/tagData';
-import Tags from '../../components/Tags';
 
 
-const icons = creativeIconPics.map(icon => icon.url);
-
 export default function Couseling() {
```
Comment (drteresavasquez) [inline pages/resources/counseling.js]: Looks like we have a misspelling here.

**Review by drteresavasquez — CHANGES_REQUESTED** [review]

File: pages/resources/counseling.js (hunk 1 above)
Comment (bmgdevelopment) [reply pages/resources/counseling.js]: Thank you! I'll edit and repush

Comment (drteresavasquez) [reply pages/resources/counseling.js]: @bmgdevelopment make sure to pull down development into this branch as well. There seem to be some conflicts

Comment (bmgdevelopment) [reply pages/resources/counseling.js]: just seeing this, I will do a pull

**Review by drteresavasquez — APPROVED** [review]
👍🏼 NOICE!

**Pull request merged**